packages/webviz-core/src/panels/ImageView/decodings.js

```diff
@@ -114,3 +114,16 @@ export function decodeRGGB(rggb: Uint8Array, width: number, height: number, outp
     }
   }
 }
+
+export function decodeMono8(mono8: Uint8Array, width: number, height: number, output: Uint8ClampedArray) {
+  let inIdx = 0;
+  let outIdx = 0;
+
+  for (let i = 0; i < width * height; i++) {
+    const ch = mono8[inIdx++];
```

Comment:
For `16UC1` / `mono16`, I think you would want `(data[inIdx++] << 8) | data[inIdx++]` (assuming big endian – not sure where this is documented). There should be twice as many values in the input data because it's 2 bytes per value instead of 1.

Reply:
actually, just noticed there's an `is_bigendian` on the image: http://docs.ros.org/melodic/api/sensor_msgs/html/msg/Image.html

Reply:
you could also use DataView like we do for 32FC1

Reply:
does that mean we should support is_bigendian = false for all the encodings?

Reply:
Yeah probably that would be ideal, for example this `true` should be `!is_bigendian`: https://github.com/cruise-automation/webviz/blob/f1d751aa8320e33f4a501a2e4b3a37851423a268/packages/webviz-core/src/panels/ImageView/decodings.js#L59

it doesn't affect the 8-bit encodings though.